Modules/_asynciomodule.c

```diff
@@ -1587,6 +1692,19 @@ FutureIter_am_send(futureiterobject *it,
     return PYGEN_ERROR;
 }
 
+static PySendResult
+FutureIter_am_send(futureiterobject *it,
+                   PyObject *Py_UNUSED(arg),
+                   PyObject **result)
+{
+    PySendResult res;
+    Py_BEGIN_CRITICAL_SECTION2(it, it->future);
+    res = FutureIter_am_send_lock_held(it, Py_None, result);
```

Comment:
If `arg` (`Py_None`) is unused, maybe remove it?

Reply:
done